app/channel-settings/server/functions/saveReactWhenReadOnly.js

```diff
@@ -1,12 +1,21 @@
 import { Meteor } from 'meteor/meteor';
 import { Match } from 'meteor/check';
 
-import { Rooms } from '../../../models';
+import { Rooms, Messages } from '../../../models';
 
-export const saveReactWhenReadOnly = function(rid, allowReact) {
+export const saveReactWhenReadOnly = function(rid, allowReact, user, sendMessage = true) {
```

Comment:
why you need this `sendMessage`?

Reply:
Some other similar functions (eg `saveRoomAnnouncement`, `saveRoomType`) in this directory use this parameter, so I decided to include it here too. Anyways, if we think it'd be better to remove it, I can do it 😄

Reply:
hmmmm I think it's worth checking if it makes any difference in this case, if it doesn't I think it's better to just leave as it is

Reply:
In the other functions that contain the `sendMessage` parameter, it is always omitted from their calls. So, I think this isn't relevant at all 🤷‍♂️ . Guess we can just leave it as it is, then